tests/Feature/Conversations/Messages/IndexTest.php

```diff
@@ -81,6 +89,8 @@ private function prepare(): array
         // Add user as participant.
         $conversation->users()->attach($user);
 
+        MediaManager::upload(Message::find(1), $this->base64);
```

Comment:
Bad practice. Just use a factory for `Media`. You don't have to upload a real file to test this.